templates/testing/index.html

```diff
@@ -0,0 +1,215 @@
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2>What’s under the hood</h2>
+      <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android game streaming from the cloud.</h4>
+    </div>
+  </div>
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h3>Android in containers</h3>
+      <p>Unlike Android emulators that make use of virtual machines, Anbox Cloud leverages system containers through <a class="p-link--external" href="https://linuxcontainers.org/">LXD</a>, the next generation system container manager. Offering a user experience similar to virtual machines but using lightweight Linux containers instead, it is  managed via a REST API. You can take advantage of automation-friendly interfaces to streamline your CI/CD testing pipeline.</p>
+    </div>
+    <div class="col-5 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ac9a0e00-lxd_primary.svg",
+        alt="",
+        width="229",
+        height="75",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h3>Software orchestration</h3>
+      <p>Emulating Android applications in thousands of virtual mobile devices mobile simultaneously requires orchestration and monitoring of several software components. This involves streaming servers, image repositories, mobile applications, a dashboard application for management and <a href="https://anbox-cloud.io/docs/overview">more</a>. Thanks to <a href="https://jaas.ai/">Juju</a> and <a href="https://maas.io/">MAAS</a>, you can deploy your stack automatically anywhere.</p>
+    </div>
+    <div class="col-5 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/48a8eddf-juju_black-orange_hex.svg",
+        alt="",
+        width="208",
+        height="75",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+        }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-6 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+        alt="",
+        width="289",
+        height="205",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+    <div class="col-5">
+      <h3>Automate Android app testing</h3>
+      <p>Get in touch with Canonical to explore avenues to modernise your Android application testing infrastructure.</p>
+      <button href="/contact-us"  class="p-button--positive">Get in touch</button>
```

Comment:
Buttons don't have hrefs
```suggestion
      <a href="/contact-us"  class="p-button--positive">Get in touch</a>
```